repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/search-main.ftl

```diff
@@ -0,0 +1,29 @@
+<#include "../include/imports.ftl">
+
+<#-- @ftlvariable name="query" type="java.lang.String" -->
+<#-- @ftlvariable name="pageable" type="org.onehippo.cms7.essentials.components.paging.Pageable" -->
+<#if pageable??>
+  <#if pageable.total == 0>
+    <h3>No results for: ${query?html}</h3>
+  <#else>
+    <div>
+      <#list pageable.items as item>
+        <#if item.title??>
+          <#assign linkName=item.title/>
+        <#else>
+          <#assign linkName=item.localizedName/>
+        </#if>
+        <article>
+          <@hst.cmseditlink hippobean=item/>
+          <@hst.link var="link" hippobean=item />
+          <h3><a href="${link}">${linkName?html}</a></h3>
+        </article>
+      </#list>
+      <#if cparam.showPagination>
+        <#include "../include/pagination.ftl">
+      </#if>
+    </div>
+  </#if>
+<#else>
+  <h3>Please fill in a search term.</h3>
+</#if>
```

Comment:
This file is not used for search results, so could be removed. The search results page is located at freemarker/publication/searchresults.ftl

Reply:
I'm on it...